balancer/weightedroundrobin/balancer.go

```diff
@@ -45,6 +47,43 @@ import (
 // Name is the name of the weighted round robin balancer.
 const Name = "weighted_round_robin"
 
+var (
+	rrFallbackHandle = estats.RegisterInt64Count(estats.MetricDescriptor{
```

Comment:
Nit: `rrFallbackMetric` or `Instrument` or something more concrete?  `Handle` doesn't describe it.

Reply:
Chose the former, xxxMetric for all 4 handles.